src/main/java/org/opentripplanner/netex/mapping/StopMapper.java

```diff
@@ -44,23 +51,52 @@ Stop mapQuayToStop(
       return null;
     }
 
+    var wheelchairBoarding = wheelChairBoardingFromQuay(quay, parentStation);
+
     Stop stop = new Stop(
-        idFactory.createId(quay.getId()),
-        parentStation.getName(),
-        quay.getPublicCode(),
-        quay.getDescription() != null ? quay.getDescription().getValue() : null,
-        WgsCoordinateMapper.mapToDomain(quay.getCentroid()),
-        null,
-        null,
-        null,
-        fareZones,
-        null,
-        null,
-        transitMode.first,
-        transitMode.second
+            idFactory.createId(quay.getId()),
+            parentStation.getName(),
+            quay.getPublicCode(),
+            quay.getDescription() != null ? quay.getDescription().getValue() : null,
+            WgsCoordinateMapper.mapToDomain(quay.getCentroid()),
+            wheelchairBoarding,
+            null,
+            null,
+            fareZones,
+            null,
+            null,
+            transitMode.first,
+            transitMode.second
     );
+
     stop.setParentStation(parentStation);
 
     return stop;
   }
+
+  /**
+   * Get WheelChairBoarding from Quay and parent Station.
+   *
+   * @param quay          NeTEx quay could contain information about accessability
+   * @param parentStation Used to get a default WheelChairBoarding if not found from Quay.
+   * @return not null value with default NO_INFORMATION if nothing defined in quay or
+   * parentStation.
+   */
+  private WheelChairBoarding wheelChairBoardingFromQuay(Quay quay, Station parentStation) {
+
+    var defaultWheelChairBoarding = WheelChairBoarding.NO_INFORMATION;
+    var stopPlaceId = parentStation.getId();
+    var stopPlace = stopPlaceIndex.lookupLastVersionById(stopPlaceId.getId());
```

Comment:
@TimMickelsonSkanetrafiken I know that this was already merged, but I think there is an error here, as the `lookupLastVersionById` might return a different version that what the `parentStation` is based on. Previously our strategy was to pass down the information in the call chain from the original stop place, and not to do a second lookup from the index. You can look at `fareZones` and `transitMode` as examples.

Reply:
@hannesj I've created #3965 please review and let me know.